admin-data-explorer-web/src/main/scala/net/wiringbits/webapp/utils/ui/web/components/EditGuesser.scala

```diff
@@ -1,27 +1,41 @@
 package net.wiringbits.webapp.utils.ui.web.components
 
-import japgolly.scalajs.react.vdom.html_<^.VdomNode
+import japgolly.scalajs.react.vdom.html_<^._
 import net.wiringbits.webapp.utils.api.models.AdminGetTables
 import net.wiringbits.webapp.utils.ui.web.facades.reactadmin._
-import net.wiringbits.webapp.utils.ui.web.models.ColumnType
+import net.wiringbits.webapp.utils.ui.web.models.{ColumnType, TableAction}
 import net.wiringbits.webapp.utils.ui.web.utils.ResponseGuesser
+import org.scalajs.dom
 
 object EditGuesser {
 
-  def apply(response: AdminGetTables.Response.DatabaseTable) = {
+  def apply(response: AdminGetTables.Response.DatabaseTable, tableActionsMaybe: Option[TableAction]) = {
```

Comment:
I'd update `tableActionsMaybe` to `tableActions: List[TableAction]` because we could easily need many custom actions

Reply:
`TableAction` has a `List[Action]`, this already allows to have many custom actions on the same table